avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -4,8 +4,8 @@
 
 
 [GENERAL]
-# Path to samos executable and AK_Attributes files
-samosAT = ./samosAT -files files/AK_Attributes/
+# Path to com1DFA executable and AK_Attributes files
+com1Exe = ./com1DFA -files files/AK_Attributes/
```

Comment:
I tried to rename the exe com1DFA but since there is already a com1DFA directory, this is not possible.
I had to name it com1DFE.exe for it to work.